src/alchemlyb/workflows/abfe.py

```diff
@@ -0,0 +1,665 @@
+import os
+from os.path import join
+from glob import glob
+import pandas as pd
+import numpy as np
+import logging
+import matplotlib.pyplot as plt
+
+from .base import WorkflowBase
+from ..parsing import gmx, amber
+from ..preprocessing.subsampling import decorrelate_dhdl, decorrelate_u_nk
+from ..estimators import BAR, TI
+from ..estimators import AutoMBAR as MBAR
+from ..visualisation import (plot_mbar_overlap_matrix, plot_ti_dhdl,
+                             plot_dF_state, plot_convergence)
+from ..postprocessors.units import get_unit_converter
+from ..convergence import forward_backward_convergence
+from .. import concat
+from .. import __version__
+
+
+class ABFE(WorkflowBase):
+    '''Alchemical Analysis style automatic workflow.
+
+    Parameters
+    ----------
+    units : str
+        The unit used for printing and plotting results. {'kcal/mol', 'kJ/mol',
+        'kT'}. Default: 'kT'.
+    software : str
+        The software used for generating input. {'GROMACS', 'AMBER'}
+    dir : str
+        Directory in which data files are stored. Default: os.path.curdir.
```

Comment:
If these are input files then make this clear. What do the files need to look like. Does it have to be a flat directory? How are the files found by the workflow? This is really important for anyone wanting to use it: what does the code expect to find?

Reply:
Somewhere you need to describe *how* inputfiles are detected and sorted, i.e., how are dir, prefix, suffix used.